boards/arm/mm_feather/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2021 MADMACHINE LIMITED
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_library()
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+zephyr_library_sources(pinmux.c)
+zephyr_sources_ifdef(CONFIG_BOOT_FLEXSPI_NOR mmfeather_flexspi_nor_config.c)
```

Comment:
Can you update this PR similar to #37142?

Reply:
Done, Please review, Thanks!